reflex/custom_components/custom_components.py

```diff
@@ -694,11 +694,11 @@ def _collect_details_for_gallery():
             console.error(
                 f"{package_name} is owned by another user. Unable to update the information for it."
             )
-            raise click.exceptions.Exit(code=1)
+            raise SystemExit(1)
         response.raise_for_status()
     except httpx.HTTPError as he:
         console.error(f"Unable to complete request due to {he}.")
-        raise click.exceptions.Exit(code=1) from he
+        raise SystemExit(1) from None
```

Comment:
wouldn't `raise SystemExit(1)` suffice here if we didn't want to chain the exception?